specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/examples/PrivateLinkResourcesList.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2020-09-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "management",
+            "type": "Microsoft.ContainerService/managedClusters/privateLinkResources",
+            "groupId": "management",
+            "requiredMembers": [
```

Comment:
missing requiredZoneNames property
NRP Spec section 7.5

Reply:
This is implemented with spec [Private Link for AKS](https://microsoft.sharepoint.com/:w:/r/teams/WAG/AzureNetworking/_layouts/15/Doc.aspx?sourcedoc=%7B367205F4-E90D-4000-837A-664485272FE4%7D&file=Private%20Link%20NRP%20API%20Spec%20-%20AKS.dotx&action=default&mobileredirect=true&cid=63CC9A6E-8BD8-4556-8C00-576B5791D534&wdLOR=c19047B8F-9921-43B5-9C2F-DE9EF3F7C0B0), not the 1st party spec. `requiredZoneNames `  is not used there.

Reply:
Sounds good, let's take it for next iteration once supported